.github/workflows/code-checks.yml

```diff
@@ -43,8 +43,9 @@ jobs:
         env:
           FREEZE_REQUIREMENTS: 1
         run: |
-          pip install -e '.[all]' -r requirements/typing.txt
+          # TODO: investigate hanging installation with app sub-package
+          pip install -e '.[pytorch-all,fabric-all]' -r requirements/typing.txt
           pip list
 
       - name: Check typing
-        run: mypy
+        run: mypy --no-warn-unused-ignores
```

Comment:
I hope you plan to revert this change. Warning on unused ignores is what notifies us to update the code once mypy is upgraded to a better version or dependencies improve their typing annotations.

We should not ignore these in general